Resources/Textures/Logo/icon-license.txt

```diff
@@ -1,14 +1,13 @@
-The files below are licensed under CC-BY-SA 4.0 by the author who can be found at https://leonardo-dabepis.tumblr.com/
+The files below are licensed under CC-BY-SA 4.0 by the author who won't be found, but is Lyndomen
 
 icon.ico
-icon.svg
+allti.xcf
```

Comment:
what the fuck is allti.xcf

Reply:
ok but it's not in the scope of this PR so doesn't need to be included in the license